pkg/helm/schema/helm.json

```diff
@@ -0,0 +1,234 @@
+              "default": false
+            },
+            "set": {
+              "type": "object",
+              "additionalProperties": {
+                "type": "string"
+              }
+            },
+            "values": {
+              "type": "array",
+              "items": {
+                "type": "string"
+              }
+            },
+            "resetValues": {
+              "type": "boolean",
+              "default": false
+            },
+            "reuseValues": {
+              "type": "boolean",
+              "default": false
+            },
+            "outputs": {
+              "$ref": "http://schema.porter.sh/mixins/helm.json#/definitions/outputs"
+            }
+          },
+          "additionalProperties": false,
+          "required": [
+            "name",
+            "description",
+            "chart"
+          ]
+        }
+      },
+      "required": [
+        "helm"
+      ]
+    },
+    "statusStep": {
+      "type": "object",
+      "properties": {
+        "helm": {
+          "type": "object",
+          "properties": {
+            "description": {
+              "$ref": "http://schema.porter.sh/mixins/helm.json#/definitions/stepDescription"
+            },
+            "releases": {
+              "type": "array",
+              "items": {
+                "type": "string",
+                "minItems": 1
+              }
+            }
+          },
+          "additionalProperties": false,
+          "required": [
+            "description",
+            "releases"
+          ]
+        }
+      },
+      "required": [
+        "helm"
+      ]
+    },
+    "uninstallStep": {
+      "type": "object",
+      "properties": {
+        "helm": {
+          "type": "object",
+          "properties": {
+            "description": {
+              "$ref": "http://schema.porter.sh/mixins/helm.json#/definitions/stepDescription"
+            },
+            "releases": {
+              "type": "array",
+              "items": {
+                "type": "string"
+              },
+              "minItems": 1
+            },
+            "purge": {
+              "type": "boolean",
+              "default": false
+            }
+          },
+          "additionalProperties": false,
+          "required": [
+            "description",
+            "releases"
+          ]
+        }
+      },
+      "required": [
+        "helm"
+      ]
+    },
+    "stepDescription": {
+      "type": "string",
+      "minLength": 1
+    },
+    "outputs": {
+      "type": "array",
+      "items": {
+        "type": "object",
+        "properties": {
+          "name": {
+            "type": "string"
+          },
+          "secret": {
+            "type": "string"
+          },
+          "key": {
+            "type": "string"
+          }
+        },
+        "additionalProperties": false,
+        "required": ["name", "secret", "key"]
+      }
+    }
+  },
+  "type": "object",
+  "properties": {
```

Comment:
This is important for the other mixins. I messed up when I originally did the schema (and by default a json schema passes any document given to it, so I didn't notice). The root json object must have a type and properties. "install" etc cannot be at the root.

I've fixed that here, but I need to go back and fix porter to handle this properly when merging schema. in https://github.com/deislabs/porter/pull/212